api/models/DetaineeEvent.js

```diff
@@ -0,0 +1,52 @@
+"use strict";
+
+var LinkingModels = require('sails-linking-models');
+var ModelHelpers = require('../lib/ModelHelpers');
+
+const model = {
+  schema: true,
+  autoCreatedAt: true,
+  autoUpdatedAt: true,
+  attributes: {
+    id: {
```

Comment:
you might need to add:
`autoPK:false`
to the model, sails might try and make the id the public key - which is its default behaviour.
in addition is it 'ok' to not have a public key - or anyway to identify an individual row?